examples/resources/plugins/teleport-slack.toml

```diff
@@ -1,16 +1,40 @@
-# example slack plugin configuration TOML file
+# Copied from https://github.com/gravitational/teleport-plugins/blob/master/access/slack/example_config.toml
```

Comment:
Would it be possible to reference https://github.com/gravitational/teleport-plugins/blob/master/access/slack/example_config.toml directly in our docs? Or will it only render if we have the file locally?

Reply:
The remark plugin that evaluates partials replaces the `(!<path>!)` by [reading a local file](https://github.com/gravitational/docs/blob/1a972bbb3117d1845794b3a89a48cebd14973396/server/remark-includes.ts#L53), so you'll need to use a copy unfortunately.